### PR TITLE
Prefix dataset with project id

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: bigquery_etl/view/validate_views.py [hunk 1]
```diff
@@ -22,8 +22,6 @@
     "sql/moz-fx-data-shared-prod/telemetry/clients_daily_scalar_aggregates_v1/view.sql",
     "sql/moz-fx-data-shared-prod/telemetry/clients_histogram_aggregates_v1/view.sql",
     "sql/moz-fx-data-shared-prod/telemetry/clients_probe_processes/view.sql",
-    # Java errors
-    "sql/moz-fx-data-shared-prod/firefox_accounts_derived/funnel_events_source/view.sql",
```
Comment (akkomar) [inline bigquery_etl/view/validate_views.py:26]: Why is this needed? This PR seems to be passing `validate-views` test.

Comment (fbertsch) [reply bigquery_etl/view/validate_views.py:26]: Hah, it was removed. Yesterday it was giving me java errors.

Comment (akkomar) [reply bigquery_etl/view/validate_views.py:26]: Hah, I suck at reading diffs.

**Review by akkomar — APPROVED** [review]

**Comment by fbertsch** [comment]
Thanks @akkomar!
